newrelic.js

```diff
@@ -1,3 +1,8 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
I don't think we want to be adding a copyright to the template file we are asking customers to copy-paste into their applications. May need to add to the exceptions list.

Reply:
ah, good point. I can exclude this file